packages/app-mobile/components/Modal.tsx

```
@@ -185,6 +185,7 @@ const ModalElement: React.FC<ModalElementProps> = ({
 	</View>;
 
 	const extraScrollViewProps = (typeof scrollOverflow === 'object' ? scrollOverflow : {});
+	const keyBoardAvoidingViewEnabled=(Platform.OS === 'android' && Platform.Version<34) || Platform.OS === 'ios';
```

ESLint

[error] Operator '=' must be spaced.

(space-infix-ops)